include/envoy/config/subscription.h

```diff
@@ -61,9 +61,9 @@ template <class ResourceType> class Subscription {
   virtual void updateResources(const std::vector<std::string>& resources) PURE;
 
   /**
-   * @return version info from last accepted onConfigUpdate.
+   * @return std::string version info from last accepted onConfigUpdate.
```

Comment:
Could leave a trail of TODO (or put a TODO in some central place) to clean this up when we converge `string` implementations.

Reply:
To reduce the sheer quantity of TODO trail I'll put them in the base classes only, I think this is sufficient.

Reply:
SGTM